src/modules/project/pages1/projectDashboard/views/promote/ProjectDashboardPromote.tsx

```diff
@@ -0,0 +1,483 @@
+            ? editState[userId]?.payoutRate ?? ''
+            : (ambassadorRow.node?.payoutRate ?? 0) * PAYOUT_RATE_SCALE
+
+          return (
+            // Ensure right alignment for numeric/input data
+            <HStack w="full" justify="flex-start">
+              {isEditing ? (
+                <InputGroup size="sm" maxW="100px">
+                  {' '}
+                  {/* Adjust width as needed */}
+                  <Input
+                    type="number"
+                    value={currentRate}
+                    onChange={(e) => handleRateChange(userId, e.target.value)}
+                    textAlign="right"
+                    isDisabled={updateAmbassadorLoading}
+                    step="0.01"
+                  />
+                  <InputRightElement>%</InputRightElement>
+                </InputGroup>
+              ) : (
+                <Body size="sm">{currentRate}%</Body>
+              )}
+            </HStack>
+          )
+        },
+        isMobile,
+        colSpan: 2,
+      },
+      {
+        header: t('Actions'),
+        key: 'actions',
+        render(ambassadorRow: AmbassadorEdge) {
+          const userId = ambassadorRow.node?.user?.id
+          if (!userId) return null
+          const isEditing = editState[userId]?.isEditing ?? false
+
+          return (
+            // Ensure center alignment for actions
+            <HStack w="full" justify="center">
+              {isEditing ? (
+                <HStack>
+                  <IconButton
+                    icon={<Icon as={PiCheck} />}
+                    aria-label={t('Save')}
+                    size="sm"
+                    variant="ghost"
+                    colorScheme="primary1"
+                    onClick={() => handleSaveEdit(userId)}
+                    isLoading={updateAmbassadorLoading && editState[userId]?.isEditing}
+                    isDisabled={updateAmbassadorLoading}
+                  />
+                  <IconButton
+                    icon={<Icon as={PiX} />}
+                    aria-label={t('Cancel')}
+                    size="sm"
+                    variant="ghost"
+                    onClick={() => handleCancelEdit(userId)}
+                    isDisabled={updateAmbassadorLoading}
+                  />
+                </HStack>
+              ) : (
+                <IconButton
+                  icon={<Icon as={PiPencil} />}
+                  aria-label={t('Edit')}
+                  size="sm"
+                  variant="ghost"
+                  onClick={() => handleEditClick(userId)}
+                  isDisabled={updateAmbassadorLoading}
+                />
+              )}
+            </HStack>
+          )
+        },
+        isMobile,
+        colSpan: 1,
+      },
+      ...(isMobile
+        ? [
+            {
+              header: '',
+              key: 'dropdown',
+              colSpan: 1,
+              isMobile,
+            },
+          ]
+        : []),
```

Comment:
we can potentialluy make this simple, if we make the other change I suggested inside the TableAccordion.

Reply:
I can check if you pushed the changes?